providers/aws-s3/src/main/java/org/jclouds/aws/s3/blobstore/AWSS3BlobRequestSigner.java

```diff
@@ -56,7 +56,7 @@
    public AWSS3BlobRequestSigner(RestAnnotationProcessor processor, BlobToObject blobToObject,
          BlobToHttpGetOptions blob2HttpGetOptions, Class<AWSS3Client> interfaceClass,
          @org.jclouds.location.Provider Supplier<Credentials> credentials,
-         RequestAuthorizeSignature authSigner, @TimeStamp Provider<String> timeStampProvider,
+         RequestAuthorizeSignatureV2 authSigner, @TimeStamp Provider<String> timeStampProvider,
```

Comment:
sorry, this is an wrong.. BlobRequestSigner use to generating pre-signed URLs, [Share an Object with Others](http://docs.aws.amazon.com/AmazonS3/latest/dev/ShareObjectPreSignedURL.html)
AWSS3BlobRequestSigner is use for Signature V2
I'll impl AWSS3BlobRequestSignerV4 for Signature V4

[sigv4-query-string-auth](http://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-query-string-auth.html)